yarn.lock

```diff
@@ -8436,13 +8436,20 @@ __metadata:
   languageName: node
   linkType: hard
 
-"discord-api-types@npm:^0.37.27, discord-api-types@npm:^0.37.28":
+"discord-api-types@npm:^0.37.27":
   version: 0.37.28
   resolution: "discord-api-types@npm:0.37.28"
   checksum: c033b9c82d5ecb3f273d563db578e21e982a35d106cc66e7ea42d7ef0a79817ca43badd3aade8a5ddb491bc8679ed08b1bcccc105e64ecd1ea5a53a11c2b1f8b
   languageName: node
   linkType: hard
 
+"discord-api-types@npm:^0.37.30":
```

Comment:
Small nit, can you dedupe this? `yarn dedupe discord-api-types` should suffice.

Otherwise manually changing this by adding `"discord-api-types@npm:^0.37.27", ` before this YAML key and removing the lines 8439-8444 will also do the trick.